_test_yaml/test/ensure_build_test.dart

```diff
@@ -4,6 +4,8 @@
 
 @TestOn('vm')
 @Tags(['presubmit-only'])
+library test;
```

Comment:
what is up with these?

Reply:
We don't want dangling annotations. These are annotating the LIBRARY.

Reply:
oh I see, it does make things more stable I guess so sgtm